.github/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+### Pull Request checklist
+
+#### All query authors
+
+- [ ] Add a change note if necessary. See [the documentation](https://github.com/github/codeql/blob/main/docs/change-notes.md) in this repository.
```

Comment:
Inconsistent style. I'm not going to discuss which of the two styles that is preferable.

```suggestion
- [ ] Change notes are added if necessary. See [the documentation](https://github.com/github/codeql/blob/main/docs/change-notes.md) in this repository.
```

This matches the declarative style of the other bullets, especially "QL tests are added..." one.